docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -6,7 +6,7 @@ Cause Models in GBD
 
 What is a cause?
 ----------------
-A cause in the GBD is something that is responsible for death, disease, or disability. 
+A cause in the GBD is something that is responsible for death, disease, injury or disability. 
```

Comment:
This seems slightly confusing to me, in terms of being clear with our definitions. The way I think of it is that a cause is something that is responsible for death or disability, and the two categories of causes are diseases and injuries.

In other words, the cause **is** the disease or injury, and these are the things responsible for death and or disability. Does that match other people's understanding?